packages/core/src/internal/services/i18n.service.ts

```diff
@@ -0,0 +1,223 @@
+    nextDecade: string;
+    currentDecade: string;
+    selectMonthText: string;
+    selectYearText: string;
+    daypickerSRCurrentMonthPhrase: string;
+    daypickerSRCurrentYearPhrase: string;
+    daypickerSRCurrentDecadePhrase: string;
+  };
+  dropdown: {
+    open: string;
+  };
+  file: {
+    browse: string;
+    files: string;
+    removeFile: string;
+  };
+  forms: {
+    formErrorSummary: string;
+  };
+  modal: {
+    closeButtonAriaLabel: string;
+    contentStart: string;
+    contentEnd: string;
+  };
+  overflow: {
+    more: string;
+  };
+  pagination: {
+    previous: string;
+    next: string;
+    current: string;
+  };
+  signpost: {
+    toggle: string;
+    close: string;
+  };
+  stackview: {
+    valueChanged: string;
+  };
+  timeline: {
+    stepNotStarted: string;
+    stepCurrent: string;
+    stepSuccess: string;
+    stepError: string;
+    stepProcessing: string;
+  };
+  verticalnav: {
+    toggle: string;
+    groupToggle: string;
+  };
+}
+
+export const componentStringsDefault = {
+  alert: {
+    closeButtonAriaLabel: 'Close',
+    loading: 'Loading',
+    info: 'Info',
+    success: 'Success',
+    warning: 'Warning',
+    danger: 'Error',
+  },
+  datagrid: {
+    rowActions: 'Available actions',
+    pickColumns: 'Show or hide columns',
+    showColumns: 'Show Columns',
+    sortColumn: 'Sort Column',
+    firstPage: 'First Page',
+    lastPage: 'Last Page',
+    nextPage: 'Next Page',
+    previousPage: 'Previous Page',
+    currentPage: 'Current Page',
+    totalPages: 'Total Pages',
+    minValue: 'Min value',
+    maxValue: 'Max value',
+    filterAriaLabel: 'Toggle column filter',
+    showColumnsMenuDescription: 'Show or hide columns menu',
+    allColumnsSelected: 'All columns selected',
+    detailPaneStart: 'Start of row details',
+    detailPaneEnd: 'End of row details',
+    singleSelectionAriaLabel: 'Single selection header',
+    singleActionableAriaLabel: 'Single actionable header',
+    detailExpandableAriaLabel: 'Toggle more row content',
+    expandableBeginningOf: 'Beginning of',
+    expandableEndOf: 'End of',
+    expandableRowContent: 'Expandable row content',
+    expandableRowsHelperText: `Screen reader table commands may not work for viewing expanded content, please use your screen reader's browse mode to read the content exposed by this button`,
+  },
+  datepicker: {
+    toggle: 'Toggle',
+    previousMonth: 'Previous month',
+    currentMonth: 'Current month',
+    nextMonth: 'Next month',
+    previousDecade: 'Previous decade',
+    nextDecade: 'Next decade',
+    currentDecade: 'Current decade',
+    selectMonthText: 'Select month, the current month is {CALENDAR_MONTH}',
+    selectYearText: 'Select year, the current year is {CALENDAR_YEAR}',
+    daypickerSRCurrentMonthPhrase: 'The current month is {CURRENT_MONTH}',
+    daypickerSRCurrentYearPhrase: 'The current year is {CURRENT_YEAR}',
+    daypickerSRCurrentDecadePhrase: 'The current decade is {DECADE_RANGE}',
+  },
```

Comment:
Should I remove all the strings that were carried over from the Angular library and re-add them as needed we implement the components? In particular, the datepicker has these string values where a part of the value is meant to be dynamic. I don't believe we'll need this behavior with this I18nService, so at the very least I expect these values to change. Some of these keys might go away too depending on how we design / implement future components.

Reply:
That sounds good. Re-adding them as needed will give us time to review/audit them as we go so we don't include anything that isn't needed.